src/tests/ftest/cart/utest/utest_portnumber.c

```diff
@@ -193,7 +193,7 @@ static void
 test_port_tcp(void **state)
 {
 	d_setenv("OFI_INTERFACE", "lo", 1);
-	d_setenv("CRT_PHY_ADDR_STR", "ofi+tcp;ofi_rxm", 1);
+	d_setenv("CRT_PHY_ADDR_STR", "ofi+tcp", 1);
 	run_test_fork(state);
 }
 
```

Comment:
there appears to be already an ofi+tcp test below. i tihnk instead we should keep this test as ofi+tcp and ofi+tcp;ofi_rxm  and test both